### PR TITLE
Updated command reference after unescaping angle brackets

### PR DESCRIPTION
**Pull request by bravo-kernel**
Update generated by alt3 module 1.0.32.

Solves the angle bracket issues:

- `\<` and `\>` are converted to `&lt;` and `%gt;` when not inside a code block
- `\<` and `\>` are converted to `<` and `>` when inside a code block
- `\<` and `\>` are converted to `<` and `>` when inside inline code
- Adds alt3 module version to sidebar config

Please note:

- Updated indentation for `Call Arguments` in `New-MockObject.mdx` now matches [Pester source  code](https://github.com/pester/Pester/blob/main/src/functions/New-MockObject.ps1#L57)

### CODE REVIEW

**Comment by bravo-kernel** [comment]
Aha, seems the brackets should be translated to `&lt;` and `&gtl`. 

Pretty dumb mistake. Time to integrate local website testing into the flow 🤦‍♀️

**Comment by bravo-kernel** [comment]
@fflaten please check, you can now use angle brackets anywhere.

File: docs/commands/New-MockObject.mdx [hunk 1]
```diff
@@ -70,14 +70,14 @@ $obj._Kill
 
 Call Arguments
 ---- ---------
-   1 {}
-   2 {True}
-   3 {False}
+1 {}
+2 {True}
+3 {False}
```
Comment (fflaten) [inline docs/commands/New-MockObject.mdx:75]:
```suggestion
   1 {}
   2 {True}
   3 {False}
```

File: docs/commands/New-MockObject.mdx [hunk 2]
```diff
@@ -70,14 +70,14 @@ $obj._Kill
...
 ```
 
 Create a mock of a process-object and mocks the object's Kill()-method.
 The mocked method will keep a history
-of any call and the associated arguments in a key named _MethodName
+of any call and the associated arguments in a key named &lt;_&lt;MethodName
```
Comment (fflaten) [inline docs/commands/New-MockObject.mdx:80]:
```suggestion
of any call and the associated arguments in a key named _MethodName
```

**Review by fflaten — COMMENTED** [review]
Awesome! Just a few nits as I did not remember my own [note](https://github.com/pester/docs/pull/247#discussion_r1066918040) about fixing source 😄

**Comment by bravo-kernel** [comment]
Perfect, so you are fixing those two in the Pester source correct? If so, I will merge this.

**Comment by fflaten** [comment]
> Perfect, so you are fixing those two in the Pester source correct? If so, I will merge this.

Yes, but please commit the suggestions for now so the website is correct until new Pester-release :)

**Comment by fflaten** [comment]
@bravo-kernel: When I run `./generate-command-reference.ps1` now, it uses 1.0.32, but I get angle brackets in the MDX which causes JSX syntax-errors on build. Ex:

image

image

What am I doing wrong?

**Comment by bravo-kernel** [comment]
Nothing, I need to release a new version. Will do that right now.

**Comment by bravo-kernel** [comment]
Just released via https://github.com/alt3/Docusaurus.Powershell/pull/158, it should be published within 10 minutes. 

Usage of angle brackets is described at https://github.com/alt3/Docusaurus.Powershell/pull/157.

Also good to know... the module now creates all debugging information (per command) in `%TEMP%\Alt3.Docusaurus.Powershell`. That shoud make it more easy to reason about strange results because the raw platy-produced `.md` file is also found in that folder.

**Comment by fflaten** [comment]
That solved it - thanks! :)

**Comment by bravo-kernel** [comment]
Sweet ☺

**Pull request merged**